presto-main/src/main/java/com/facebook/presto/type/TypeJsonUtils.java

```diff
@@ -156,6 +156,7 @@ else if (type.getJavaType() == Slice.class) {
         }
     }
 
+    @Deprecated
```

Comment:
It looks like this whole class is unused, so just delete it. Or at least this method, if others are still used